k8s-deploy/Makefile

```diff
@@ -68,12 +70,12 @@ swag: swag-bin
 package: kubefate-without-swag
 	mkdir -p tmp/kubefate;
 	cp -r bin/kubefate *.yaml examples tmp/kubefate;
-	tar -czvf kubefate-k8s-${RELEASE_VERSION}.tar.gz -C tmp/ kubefate;
+	tar -czvf kubefate-k8s-${VERSION}-${ARCH}.tar.gz -C tmp/ kubefate;
```

Comment:
Your need to change the version back to release_version here.

https://github.com/FederatedAI/KubeFATE/releases/tag/v1.9.0

You can check the assets in the releases, we use the fate release version to name the Kubefate release.

v1.4.5 is like the kubefate docker image version, or the kubefate cli version.